samples/msal-angular-v2-samples/angular10-sample-app/jest.config.js

```diff
@@ -3,7 +3,7 @@ module.exports = {
     globals: {
         __PORT__: 4201,
         __STARTCMD__: "npm start -- --port 4201",
-        __TIMEOUT__: 45000
+        __TIMEOUT__: 90000
```

Comment:
Again doubling the timeout.  For things that do timeout, doesn't this increase the time to run the entire test pass?

Reply:
This would increase the time to failure if the sample fails to build/serve but should not result in longer test runs overall

Reply:
The sample failing to build is actually pretty rare, it's more common that it takes a bit longer than we've allotted here.